### ISSUE TITLE
Perm Ban is here...

### ISSUE THREAD

**Issue by Tymoc** [Ar1i/PokemonGo-Bot#3262]
So during the short time that the bot was taken down, i started to use Bluestacks player as a work around... within 5 minutes of using it, the whole thing went down and showed "network error".

I tried to then login from my phone, to find that it said, "Server can not get data"...  so I send in a support ticket to Niantic, and here is their response:

"Trainer,

We have received your appeal. Our goal is to provide a fair, fun and legitimate game experience for everyone. Use of any other third-party software or app, including but not limited to scripts/apps, applications used to falsify your location, using emulators, modified or unofficial software and/or accessing Pokémon GO clients or backends in an unauthorized manner is considered cheating.

-NianticOps"

basically this is a perm ban... sucks that it was my main account at level 32!

Now i logged into my alt account, and I can still 'see' my pokemon holding the gyms near me.  Maybe there's hope?

Either way - I just charged back all the $$ i did spend on that account.

**Comment by Midnlght** [comment]
I mean why would you bot on your main account lol? and it was expect to happen sooner or later you should have been aware of the consequences mate

**Comment by xJustx** [comment]
Thanks for this info man!
